Saule/Http/JsonApiProcessor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Web.Http.Filters;
+using Saule.Serialization;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Processes JSON API responses
+    /// </summary>
+    public class JsonApiProcessor
+    {
+        /// <summary>
+        /// process request as JSON API response
+        /// </summary>
+        /// <param name="context">context</param>
+        public static void ProcessRequest(HttpActionExecutedContext context)
+        {
+            var statusCode = (int)context.Response.StatusCode;
+            if (statusCode >= 400 && statusCode < 500)
+            {
+                // probably malformed request or not found
+                return;
+            }
+
+            var value = context.Response.Content as ObjectContent;
+
+            var config = new JsonApiConfiguration();
```

Comment:
@joukevandermaas, I couldn't figure out how to get the configuration, so I created a new one.  Is that OK?  Is there a way to get the configuration?

Reply:
That's fine, perhaps we can also create an overload that takes it as a parameter.